docs/architecture/adr-001-poap-contract.md

```diff
@@ -29,8 +29,8 @@ We will create a `POAP` contract that allows a selected `minter` (ideally anothe
 collection associated with a specific event. The contract will implement the `CW721-base`, the basic implementation of
 NFTs in CosmWasm.
 The `CW721-base` will take care of store all the information related to:
-- Associations between POAPs and attendees;
-- Associations between POAPs and metadata.
```

Comment:
Metadata does not store in cw721-base, it handled by `EventInfo` in POAP contract now.

Reply:
Nevermind, it still store metadata in cw721-base